### ISSUE TITLE
Crashes when mp3 player is connected.

### ISSUE THREAD

**Issue by realSConway** [clementine-player/Clementine#6791]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [X] I checked the issue tracker for similar issues
- [X] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [X] I tried the latest Clementine build from [here](https://builds.clementine-player.org/) (Running latest Opensuse tumbleweed release 20200902)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Opensuse tumbleweed
- Clementine version: 1.3.9

### Expected behaviour / actual behaviour
Clementine opens when I have usb plugged in

### Steps to reproduce the problem (only for bugs)
Hello experts,
I'm up against following problem. MP3 player or any other usb device connected to pc, open clementine it will flash on the screen, then close.

Running clementine from command line, following output.
```
$ clementine
13:44:01.694 INFO  main:307                         Clementine-qt5 1.3.9
13:44:01.870 DEBUG NetworkProxyFactory:49           Detected system proxy URLs: ("", "", "", "")
13:44:01.870 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fe071ffa710 "/usr/bin/clementine-tagreader" "/tmp/clementine_759303087"
13:44:01.872 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fe071ffa710 "/usr/bin/clementine-tagreader" "/tmp/clementine_664854722"
13:44:01.874 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fe071ffa710 "/usr/bin/clementine-tagreader" "/tmp/clementine_1639688294"
13:44:01.877 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fe071ffa710 "/usr/bin/clementine-tagreader" "/tmp/clementine_1600317021"
13:44:01.921 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_664854722"
13:44:01.921 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fe05c007da0 connected to "/tmp/clementine_664854722"
13:44:01.925 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_759303087"
13:44:01.925 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fe05c006820 connected to "/tmp/clementine_759303087"
13:44:01.939 DEBUG QxtGlobalShortcutBackend:32      registering
13:44:01.955 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_1639688294"
13:44:01.955 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fe05c006360 connected to "/tmp/clementine_1639688294"
13:44:01.957 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_1600317021"
13:44:01.957 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fe05c00aa50 connected to "/tmp/clementine_1600317021"
13:44:02.040 DEBUG logging:68                       _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
13:44:02.172 DEBUG MainWindow:228                   Starting
__logging_message__13:44:02.178 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
__logging_message__13:44:02.367 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
13:44:02.369 DEBUG CoverProviders:35                Registered cover provider "MusicBrainz"
13:44:02.369 DEBUG CoverProviders:35                Registered cover provider "Discogs"
13:44:02.369 DEBUG CoverProviders:35                Registered cover provider "last.fm"
13:44:02.377 DEBUG MainWindow:308                   Initialising player
13:44:02.386 WARN  PulseDeviceFinder:51             Failed to connect pulseaudio context
13:44:02.487 DEBUG MainWindow:314                   Creating models
13:44:02.548 INFO  Player:639                       Registered URL handler for "classicalradio"
13:44:02.548 DEBUG InternetModel:133                Adding internet service: "ClassicalRadio"
13:44:02.548 INFO  Player:639                       Registered URL handler for "di"
13:44:02.548 DEBUG InternetModel:133                Adding internet service: "DigitallyImported"
13:44:02.653 DEBUG InternetModel:133                Adding internet service: "Icecast"
13:44:02.656 DEBUG InternetModel:133                Adding internet service: "Jamendo"
13:44:02.657 INFO  Player:639                       Registered URL handler for "jazzradio"
13:44:02.657 DEBUG InternetModel:133                Adding internet service: "JazzRadio"
13:44:02.660 INFO  Player:639                       Registered URL handler for "magnatune"
13:44:02.661 DEBUG InternetModel:133                Adding internet service: "Magnatune"
13:44:02.663 DEBUG InternetModel:133                Adding internet service: "Podcasts"
13:44:02.664 INFO  Player:639                       Registered URL handler for "rockradio"
13:44:02.664 DEBUG InternetModel:133                Adding internet service: "RockRadio"
13:44:02.665 DEBUG InternetModel:133                Adding internet service: "SavedRadio"
13:44:02.665 INFO  Player:639                       Registered URL handler for "radiotunes"
13:44:02.665 DEBUG InternetModel:133                Adding internet service: "RadioTunes"
13:44:02.666 INFO  Player:639                       Registered URL handler for "somafm"
13:44:02.666 DEBUG InternetModel:133                Adding internet service: "SomaFM"
13:44:02.667 INFO  Player:639                       Registered URL handler for "intergalacticfm"
13:44:02.667 DEBUG InternetModel:133                Adding internet service: "Intergalactic FM"
13:44:02.673 DEBUG SpotifyService:108               Spotify system blob path: "/usr/bin/clementine-spotifyblob"
13:44:02.673 DEBUG SpotifyService:109               Spotify local blob path: "/home/sam/.config/Clementine/spotifyblob/version16-64bit/blob"
13:44:02.674 DEBUG InternetModel:133                Adding internet service: "Spotify"
13:44:02.674 INFO  Player:639                       Registered URL handler for "subsonic"
13:44:02.790 DEBUG InternetModel:133                Adding internet service: "Subsonic"
13:44:02.888 INFO  Player:639                       Registered URL handler for "box"
13:44:02.888 DEBUG InternetModel:133                Adding internet service: "Box"
13:44:02.891 INFO  Player:639                       Registered URL handler for "dropbox"
13:44:02.891 DEBUG InternetModel:133                Adding internet service: "Dropbox"
13:44:02.894 INFO  Player:639                       Registered URL handler for "googledrive"
13:44:02.894 DEBUG InternetModel:133                Adding internet service: "Google Drive"
13:44:02.897 INFO  Player:639                       Registered URL handler for "seafile"
13:44:02.897 DEBUG InternetModel:133                Adding internet service: "Seafile"
__logging_message__13:44:02.897 WARN  unknown                          libpng warning: iCCP: known incorrect sRGB profile
13:44:02.900 INFO  Player:639                       Registered URL handler for "skydrive"
13:44:02.900 DEBUG InternetModel:133                Adding internet service: "OneDrive"
13:44:02.902 DEBUG CddaLister:98                    No CD devices found
13:44:02.902 WARN  DeviceKitLister:55               Error enumerating DeviceKit-disks devices: "org.freedesktop.DBus.Error.ServiceUnknown" "The name org.freedesktop.UDisks was not provided by any .service files"
13:44:05.872 DEBUG MainWindow:332                   Creating UI
13:44:05.983 DEBUG MainWindow:881                   Creating equalizer
13:44:05.983 DEBUG MainWindow:910                   Creating now playing widget
13:44:06.025 DEBUG Playlist:173                     k_max_scrobble_point 240
13:44:06.027 DEBUG MainWindow:972                   Loading settings
13:44:06.028 INFO  NetworkRemote:83                 Network Remote deactivated
13:44:06.155 DEBUG MainWindow:2211                  command line options received
13:44:06.155 DEBUG MainWindow:2292                  false
13:44:06.155 DEBUG MainWindow:1061                  Started
13:44:06.280 INFO  DeviceManager:172                Device added from database:  "SD_CARD"
13:44:06.280 INFO  DeviceManager:163                Found existing device:  "SD_CARD"
13:44:06.280 INFO  DeviceManager:355                Device added: "Udisks2/104552D5428975/SONY/WALKMAN/127848677376/6335-3034"
13:44:06.280 INFO  DeviceManager:355                Device added: "Udisks2/104552D5428975/SONY/WALKMAN/13167493120/78C3-19FA"
fish: “clementine” terminated by signal SIGSEGV (Address boundary error)
```

If I don't have usb plugged in, then clementine opens without issues.
```
$ clementine
14:45:58.758 INFO  main:307                         Clementine-qt5 1.3.9
14:45:58.958 DEBUG NetworkProxyFactory:49           Detected system proxy URLs: ("", "", "", "")
14:45:58.958 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fb41a7fb710 "/usr/bin/clementine-tagreader" "/tmp/clementine_-1777020721"
14:45:58.962 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fb41a7fb710 "/usr/bin/clementine-tagreader" "/tmp/clementine_-1661584478"
14:45:58.963 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fb41a7fb710 "/usr/bin/clementine-tagreader" "/tmp/clementine_-621972730"
14:45:58.983 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7fb41a7fb710 "/usr/bin/clementine-tagreader" "/tmp/clementine_-466338499"
14:45:59.014 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_-1661584478"
14:45:59.015 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fb3fc007da0 connected to "/tmp/clementine_-1661584478"
14:45:59.028 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_-1777020721"
14:45:59.028 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fb3fc0068a0 connected to "/tmp/clementine_-1777020721"
14:45:59.032 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_-621972730"
14:45:59.032 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fb3fc006360 connected to "/tmp/clementine_-621972730"
14:45:59.035 DEBUG QxtGlobalShortcutBackend:32      registering
14:45:59.058 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_-466338499"
14:45:59.059 DEBUG WorkerPool<HandlerType>:301      Worker 0x7fb3fc00aa40 connected to "/tmp/clementine_-466338499"
14:45:59.140 DEBUG logging:68                       _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
14:45:59.318 DEBUG MainWindow:228                   Starting
__logging_message__14:45:59.324 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
__logging_message__14:45:59.465 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
14:45:59.466 DEBUG CoverProviders:35                Registered cover provider "MusicBrainz"
14:45:59.466 DEBUG CoverProviders:35                Registered cover provider "Discogs"
14:45:59.466 DEBUG CoverProviders:35                Registered cover provider "last.fm"
14:45:59.473 DEBUG MainWindow:308                   Initialising player
14:45:59.487 WARN  PulseDeviceFinder:51             Failed to connect pulseaudio context
14:45:59.585 DEBUG MainWindow:314                   Creating models
14:45:59.653 INFO  Player:639                       Registered URL handler for "classicalradio"
14:45:59.653 DEBUG InternetModel:133                Adding internet service: "ClassicalRadio"
14:45:59.654 INFO  Player:639                       Registered URL handler for "di"
14:45:59.654 DEBUG InternetModel:133                Adding internet service: "DigitallyImported"
14:45:59.750 DEBUG InternetModel:133                Adding internet service: "Icecast"
14:45:59.754 DEBUG InternetModel:133                Adding internet service: "Jamendo"
14:45:59.755 INFO  Player:639                       Registered URL handler for "jazzradio"
14:45:59.755 DEBUG InternetModel:133                Adding internet service: "JazzRadio"
14:45:59.759 INFO  Player:639                       Registered URL handler for "magnatune"
14:45:59.760 DEBUG InternetModel:133                Adding internet service: "Magnatune"
14:45:59.762 DEBUG InternetModel:133                Adding internet service: "Podcasts"
14:45:59.763 INFO  Player:639                       Registered URL handler for "rockradio"
14:45:59.763 DEBUG InternetModel:133                Adding internet service: "RockRadio"
14:45:59.764 DEBUG InternetModel:133                Adding internet service: "SavedRadio"
14:45:59.764 INFO  Player:639                       Registered URL handler for "radiotunes"
14:45:59.764 DEBUG InternetModel:133                Adding internet service: "RadioTunes"
14:45:59.765 INFO  Player:639                       Registered URL handler for "somafm"
14:45:59.765 DEBUG InternetModel:133                Adding internet service: "SomaFM"
14:45:59.765 INFO  Player:639                       Registered URL handler for "intergalacticfm"
14:45:59.766 DEBUG InternetModel:133                Adding internet service: "Intergalactic FM"
14:45:59.772 DEBUG SpotifyService:108               Spotify system blob path: "/usr/bin/clementine-spotifyblob"
14:45:59.772 DEBUG SpotifyService:109               Spotify local blob path: "/home/sam/.config/Clementine/spotifyblob/version16-64bit/blob"
14:45:59.773 DEBUG InternetModel:133                Adding internet service: "Spotify"
14:45:59.773 INFO  Player:639                       Registered URL handler for "subsonic"
14:45:59.890 DEBUG InternetModel:133                Adding internet service: "Subsonic"
14:46:00.045 INFO  Player:639                       Registered URL handler for "box"
14:46:00.045 DEBUG InternetModel:133                Adding internet service: "Box"
14:46:00.051 INFO  Player:639                       Registered URL handler for "dropbox"
14:46:00.051 DEBUG InternetModel:133                Adding internet service: "Dropbox"
14:46:00.056 INFO  Player:639                       Registered URL handler for "googledrive"
14:46:00.056 DEBUG InternetModel:133                Adding internet service: "Google Drive"
14:46:00.059 INFO  Player:639                       Registered URL handler for "seafile"
14:46:00.059 DEBUG InternetModel:133                Adding internet service: "Seafile"
__logging_message__14:46:00.059 WARN  unknown                          libpng warning: iCCP: known incorrect sRGB profile
14:46:00.062 INFO  Player:639                       Registered URL handler for "skydrive"
14:46:00.062 DEBUG InternetModel:133                Adding internet service: "OneDrive"
14:46:00.064 DEBUG CddaLister:98                    No CD devices found
14:46:00.065 WARN  DeviceKitLister:55               Error enumerating DeviceKit-disks devices: "org.freedesktop.DBus.Error.ServiceUnknown" "The name org.freedesktop.UDisks was not provided by any .service files"
14:46:02.987 DEBUG MainWindow:332                   Creating UI
14:46:03.059 DEBUG MainWindow:881                   Creating equalizer
14:46:03.060 DEBUG MainWindow:910                   Creating now playing widget
14:46:03.095 DEBUG Playlist:173                     k_max_scrobble_point 240
14:46:03.097 DEBUG MainWindow:972                   Loading settings
14:46:03.098 INFO  NetworkRemote:83                 Network Remote deactivated
14:46:03.220 DEBUG MainWindow:2211                  command line options received
14:46:03.220 DEBUG MainWindow:2292                  false
14:46:03.220 DEBUG MainWindow:1061                  Started
14:46:03.333 INFO  DeviceManager:172                Device added from database:  "SD_CARD"
14:46:03.333 INFO  DeviceManager:163                Found existing device:  "SD_CARD"
14:47:26.049 DEBUG Database:604                     Starting database integrity check
14:47:26.714 DEBUG Database:665                     Starting database backup
```
I then add usb device
```
14:55:13.095 INFO  DeviceManager:355                Device added: "Gio/unmounted/140411272195232"
14:55:13.095 WARN  IconLoader:48                    Icon name is null
fish: “clementine” terminated by signal SIGSEGV (Address boundary error)
```

If anyone could point me in right direction for troubleshooting, that would be great!

**Comment by jbroadus** [comment]
It looks like SUSE still packages an old version of Clementine. There were a number of bugs around devices fixed since 1.3.1. But it looks like @jonaski has a newer version available: https://build.opensuse.org/package/show/home%3Ajonaski%3Aaudio/clementine

**Comment by realSConway** [comment]
Solved by upgrading to tuimbleweed version: 20200910
